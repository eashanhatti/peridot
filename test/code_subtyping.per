def foo: Struct{} = <struct{}>
metadef bar: Code(Struct{}) = struct{}
