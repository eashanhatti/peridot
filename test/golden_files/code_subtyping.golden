((fromList
    [(UVGlobal 1000, Just (Rigid (TypeType Obj)))],
  fromList [],
  fromList []),
 Let
   (fromList
      [MetaTerm
         (Id {unId = 0})
         (Rigid (CodeCoreType (Rigid (TypeType Obj))))
         (Rigid (CodeCoreIntro (Rigid (TypeType Obj)))),
       ObjTerm
         (Id {unId = 1})
         (Rigid (TypeType Obj))
         (CodeCoreElim
            (Rigid (CodeCoreIntro (Rigid (TypeType Obj)))))])
   (Rigid (TypeType Obj)))
